data/net/minecraft/network/chat/FilterMask.mapping

```diff
@@ -1,17 +1,17 @@
 CLASS net/minecraft/network/chat/FilterMask
 	METHOD <init> (I)V
-		ARG 1 nbits
+		ARG 1 size
 	METHOD <init> (Ljava/util/BitSet;Lnet/minecraft/network/chat/FilterMask$Type;)V
 		ARG 1 mask
 		ARG 2 type
 	METHOD apply (Ljava/lang/String;)Ljava/lang/String;
 		ARG 1 text
 	METHOD apply (Lnet/minecraft/network/chat/ChatMessageContent;)Lnet/minecraft/network/chat/Component;
-		ARG 1 chatMessageContent
+		ARG 1 content
 	METHOD read (Lnet/minecraft/network/FriendlyByteBuf;)Lnet/minecraft/network/chat/FilterMask;
 		ARG 0 buffer
 	METHOD setFiltered (I)V
-		ARG 1 bitIndex
+		ARG 1 filtered
```

Comment:
Does not look right.

Reply:
Why was this resolved?

Reply:
I changed it in Enigma but forgot to hit save. It's fixed now